conf/application-context.xml

```diff
@@ -368,41 +358,37 @@
     </bean>
 
     <bean id="authenticationService" class="org.sagebionetworks.bridge.services.AuthenticationServiceImpl">
-        <property name="stormpathClient" ref="stormpathClient" />
         <property name="distributedLockDao" ref="distributedLockDao"/>
         <property name="cacheProvider" ref="cacheProvider" />
         <property name="bridgeConfig" ref="bridgeConfig" />
-        <property name="accountEncryptionService" ref="accountEncryptionService" />
+        <property name="accountDao" ref="accountDao" />
         <property name="consentService" ref="consentService" />
         <property name="optionsService" ref="optionsService" />
+        <property name="healthCodeService" ref="healthCodeService" />
         <property name="signInValidator">
             <bean class="org.sagebionetworks.bridge.validators.SignInValidator"/>
         </property>
+        <property name="signUpValidator">
+            <bean class="org.sagebionetworks.bridge.validators.SignUpValidator"/>
+        </property>
```

Comment:
Sometimes validators need to inject a service. I just removed a case of this that went to Stormpath to check for email duplicates. If we replace Stormpath, we'd need to do something like this again. I can convert these to @Autowired so it's not an annoyance.